docs/shared/monorepo-ci-github-actions.md

```diff
@@ -30,19 +30,19 @@ jobs:
       - run: npx nx affected --target=build --parallel=3
 ```
 
-The `pr` and `main` jobs implement the CI workflow. Setting `timeout-minutes` is needed only if you have very slow tasks.
+The `pr` and `main` jobs implement the CI workflow. Setting the `timeout-minutes` is needed only if you have very slow tasks.
 
 {% callout type="note" title="Tracking the origin branch" %}
 
-If you're using this action in the context of a branch you may need to add `run: "git branch --track main origin/main"` before running `nx affected` since `origin/main` won't exist.
+If you're using this action in the context of a branch you may need to add `run: "git branch --track main origin/main"` before running the `nx affected` since `origin/main` won't exist.
```

Comment:
```suggestion
If you're using this action in the context of a branch you may need to add `run: "git branch --track main origin/main"` before running the `nx affected` command since `origin/main` won't exist.
```